client/app/templates/json-api-errors.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if errors}}
```

Comment:
Actually don't think I can.

http://emberjs.com/api/classes/Ember.Templates.helpers.html#toc_bound-template-names

that is the partial I'm using and it doesn't seem to support passing parameters...